backend/case_designer/views.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.generics import ListCreateAPIView
+from rest_framework.response import Response
+from rest_framework import status
+from .models import Bookcase, DisplayedBook
+from .paginations import BookcasePagination
+from .serializers import BookcaseSerializer
+
+class ListCreateBookcaseAPIView(ListCreateAPIView):
+    permission_classes = [IsAuthenticated]
+    serializer_class = BookcaseSerializer
+    queryset = Bookcase.objects.all()
+    pagination_class = BookcasePagination
+
+    def create(self, request, *args, **kwargs):
+        serializer = BookcaseSerializer(data=request.data)
```

Comment:
`serializer = self.get_serializer(data=request.data)` to use serializer_class variable above